spec/temporal.html

```diff
@@ -71,7 +71,7 @@ <h1>The Temporal.now Object</h1>
   </ul>
 
   <emu-clause id="sec-function-properties-of-the-temporal-now-object">
-    <h1>Function Properties of the Temporal.now Object</h1>
+    <h1>Function Properties of the Temporal.Now Object</h1>
 
     <emu-clause id="sec-temporal.now.timezone">
       <h1>Temporal.now.timeZone ( )</h1>
```

Comment:
```suggestion
      <h1>Temporal.Now.timeZone ( )</h1>
```

And you're missing a couple more below.

Reply:
Haha, this was me trying to manually undo the carnage caused by prettier. Clearly I was a bit too eager in undoing and missed a few valid changes!

For some reason I can't get the same overeager prettier behavior to reproduce anymore. Might be that my prettier plugin was auto-updated and I didn't read the release notes last time. ;-)

![image](https://user-images.githubusercontent.com/277214/125833186-e0712a53-b09f-49cd-9a9b-0aac5df2b6c0.png)